ranger-http-model/src/main/java/com/flipkart/ranger/http/response/model/RangerResponseCode.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015 Flipkart Internet Pvt. Ltd.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.flipkart.ranger.http.response.model;
+
+public enum RangerResponseCode {
+
+    SUCCESS;
```

Comment:
The way the current implementation is, error cascades are not happening properly - I am fixing them all in one place. This should be addressed in that, in a day-ish time.

Reply:
The problem with current exception is, only in the updateState, exception is explicitly thrown and checked for. In rest of the places, it is the error logging of exception and empty list etc as appropriate responses. That's why in the first place, there was no RangerResponseCode, because as they stand it is nominal and doesn't make any sense.

Reply:
Removed the RangerResponseCode. It is adding no real value and keeping it for the future, while not completely nailing down how a service provider's errors may be manifested is not a good option, imo.
    
    In the case of a finderHub though, since there could be any of the hubs that may error out (and the errors are getting logged at the point whey they error out), one response code won't cut it. I don't think it is future provisioning as well, if anything this will hinder our ability to add proper error blocks in the future when the need arises to communicate partial errors to the client.